User Guide not updated with the new commands and prefixes

AD-NAP opened the pull request:
User Guide still contain the old commands and discarded commands such as addinfo command. Table of content is also not updated.

Let's 
* update user guide with the new commands for adding and flagging
clients. 
* update table of content of UG

Note: UG Section on deleting and editing by name will be done by Vish.

This should close #69. Nice.

Commit:
User Guide not updated with the new commands

Let's update user guide with the new commands for adding and flagging
clients.

Note: UG Section on deleting and editing by name will be done by Vish.

Vishwanth2210 commented on docs/UserGuide.md:
```diff
@@ -78,48 +86,49 @@ Shows a message explaining how to access the help page.
+  Format: `edit NAME [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [i/INFO] [d/DATE] [t/TAG]…​`
 
-  * Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
+  * Edits the person named `NAME`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
```
It would be better if the second statement can be deleted as there is no more index that can be used for this command.

AD-NAP replied:
Okay got it 😄

Vishwanth2210 reviewed (changes requested):
Looks good except for that one small statement. Other than that, it looks good to be merged.

Commit:
Edit sentence to match the new edit command

Removed the sentence on edit by index. The edit command now uses name
instead of index.

Vishwanth2210 reviewed (approved):
Looks good

Vishwanth2210 committed:
Merge branch 'master' into branch-update-UG
